docs/api-reference/carto/carto-bqtiler-layer.md

```diff
@@ -1,10 +1,11 @@
 # CartoBQTilerLayer
```

Comment:
We announced the deprecation of CartoBQTilerLayer in favor of  8.5, at 8.7 we should remove it completely, maybe in a separated PR, but just to not forget about it